### ISSUE TITLE
text and image not align?

### ISSUE THREAD

**Issue by Vrchz** [zagortenay333/Harmattan#1]
I use Conky 1.9.0 in xubuntu 15.04, Theme is Cards - Fahrenheit - Comfortable
![image](https://cloud.githubusercontent.com/assets/6339105/7362297/650c4d88-ed99-11e4-9dba-a8ff1a423ce3.png)

**Comment by zagortenay333** [comment]
This is a really common problem and I still have no clue what the issue is. It must have something to do with the height of empty lines or something similar.

Just out of curiosity, could you try this config for me: http://pastebin.com/tzfXP3fK
Instead of globally defining a default font size with `xftfont Droid Sans:size=10`, I defined the font size on every line. Maybe this is a solution..

If anyone more knowledgeable with conky knows what the fix is , please share it here.
With the exception of the black box under gnome-shell, this is the biggest nuisance with Harmattan right now.

**Comment by diptaks** [comment]
Whenever I try to put other city code in the the config, it appears to be that the degree sign after the celcius turns into question marks. Tried many different way, but in vain till now. I'd appreciate if you could take a look.

**Comment by zagortenay333** [comment]
Please file individual bugs. With the conkyrc

**Comment by tryout79** [comment]
I have this issue too

**Comment by cortexx** [comment]
Hello everyone, 

This is a little bit boring to do, but you can temporally fix this alignment problem by playing with voffset value.

For example, on my .conkyrc "Numix god-mode with photos", I played with this settings : 
- In "Day Names" category at line 130 :
  `${voffset 185} -> ${voffset 172}`
- In "Conditions" category in line 150 : 
  `${voffset -170} -> ${voffset -172}`
- In "Clock and Date" category in line 162 : 
  `${voffset -120} -> ${voffset -117}`
- In "Cpu, memory, uptime, and load graph" category in line 202 : 
  `${voffset -102} -> ${voffset -104}`

It's not perfect but this seems aligned so meanwhile a bugfix, this fix do the job :)

**Comment by zagortenay333** [comment]
https://github.com/brndnmtthws/conky/issues/236

**Comment by cjmartin** [comment]
Is there a quick way to adjust the conkyrc file to account for the text offset? On my ubuntu gnome 15.04 install it looks like the text is all offset ~10px up.

Maybe the voffset tip above...

![screenshot from 2016-04-19 16-05-27](https://cloud.githubusercontent.com/assets/936/14658456/a70b729a-0648-11e6-9487-2c0177085c07.png)

**Comment by dvdmuckle** [comment]
Per suggestion of @cortexx, I changed line 130's `voffset` value for the God Mode Glass theme to 187. This seems to have fixed the issue. Not sure if this is the same for other themes/distros. Running a heavily modified version of Xubuntu 16.04.

**Comment by Zhangjiajia6** [comment]
I meet this problem too.
Have you guys fix it?
![2016-11-01 21-50-42](https://cloud.githubusercontent.com/assets/16399433/19891948/65434160-a07d-11e6-886a-008f646a6a7b.png)

**Comment by dvdmuckle** [comment]
@Zhangjiajia6 there doesn't seem to be a clear fix for this other than to just play around with `voffset` values.

**Comment by Taur10** [comment]
I ran into this on another Conky, and found that `voffset` is the solution, but the trick is to fix the `voffset` on your first element. It appears that all the other `voffset`'s appear to be relative to it, so in my case I had to increase the `voffset` by about 12, and that shifted all items into alignment.

**Comment by zagortenay333** [comment]
@Taur10 I know that, but that's not the soution. The issue is that the voffset depends on the font size, and that differs from system to system. Changing the offset by 12 might work for you but not for me.

**Comment by Taur10** [comment]
@zagortenay333 I suppose it depends on your problem, in my case, all of the text was off position towards the top of the window. By adjusting the offset on the first output text, it pushed all lines down by an equal amount, thereby realigning everything with a single change, rather than in the case of this conky, going through approximately 18 changes. As to the number, that's an estimate, YMMV, but the key here is that you can save a lot of work by changing a single value that all the others are relative to.

**Comment by zagortenay333** [comment]
Wait a minute, I just read your first comment again. No, the voffsets are not relative to the first element, they are relative to the element immediately above them. That's where the problem appears. If a line on your monitor appears too small/big you then have to readjust all the voffsets to make it work.

**Comment by Taur10** [comment]
@zagortenay333  exactly, @Zhangjiajia6 had the precise problem I had. By correcting the clock's alignment, that pushes the next element down, which pushes the next and so forth, so by adjusting the top most element, you shift the entire group down, and so long as you've maintained the designers font sizes, that one offset correction corrects the entire thing.

**Comment by zagortenay333** [comment]
No, you don't understand. You can push down everything by moving a top element like the clock, but the distance between the rows will remain unchanged.
The distance between the rows changes from distro to distro due to font size changes, but the image files remain identical. That's what causes the misalignment.

**Comment by zagortenay333** [comment]
@Taur10 and others, I tweaked the voffsets in the various form-factors according to some latest comments.

I also noticed that it works on linux mint with the new updates.. That's weird 'cause I developed harmattan on mint. Stuff must have changed in the 1.10 version.

Please check it out and tell me what distro you are using.

**Comment by Taur10** [comment]
@zagortenay333  Running it on Manjaro Xfce 16.10.3

The alignment change I made was `${color3}${voffset 186}${alignc 77}${execi 300 LANG=${template9} LC_TIME=${template9} date +%^a}${color}
` and that corrected all my alignment problems. I have done one or two other tweaks including moving it to my other monitor.

**Comment by zagortenay333** [comment]
Please check the changes I made.

**Comment by ghost** [comment]
Alignment issues seems to be fixed, but one question occurred to me though. Why is there a transparent outer area around the widget? It looks strange not mention how hard it makes to position the conky when you also have to account for the transparent border.

EDIT: Never mind, edited the .conky file to remove them.

**Comment by zagortenay333** [comment]
@szekeres2016 Oh, you selected it.. I thought that selection is what you are talking about. :laughing: 

I had to add some padding around it in case a new theme needs it.

**Comment by ghost** [comment]
Not selected it, I just draw a selection window under it, any easy way to remove it? :D

**Comment by Peloponeso31** [comment]
i resolve it playing with the code, i'm new programing so, i don't comprehend conky at all.
this is how it look's like with the fresh downloaded file:
![image](https://user-images.githubusercontent.com/32084752/30615084-4b4fb260-9d53-11e7-9c73-315388e899a1.png)
and with the modified code:
![image](https://user-images.githubusercontent.com/32084752/30615204-c608db62-9d53-11e7-8fc2-fe36926cf151.png)

i'm using manjaro btw, at 1366x768 resolution and font dpi at 76.

oh god i think i found it. is the font DPI setting that is messing all up, at 76 looks like the last image, bit with 86:
![image](https://user-images.githubusercontent.com/32084752/30615638-2273b272-9d55-11e7-8850-bee81df6c83d.png)

and 92:
![image](https://user-images.githubusercontent.com/32084752/30615689-4d0da7a4-9d55-11e7-9c21-468a8e1e8308.png)

@zagortenay333 and everyone post your DPI so we can see if is that.
Beautyful conky.

**Comment by easysid** [comment]
You can try to port the conky to lua to solve the alignment issues. lua scripts take fixed (x, y) coords so one can be sure to have the text and images line up.
